src/validations/singleCityValidator.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-unreachable-loop */
+import Joi from 'joi';
+import errorResponse from '../utils/errorResponse';
+import models from '../database/models';
+
+const { User, Accommodation } = models;
+
+const SingleTripRequestSchema = Joi.object({
+  managerId: Joi.number().required(),
+  departure_place: Joi.number().required(),
+  destination: Joi.number().required(),
+  departureDate: Joi.date().required(),
+  returnDate: Joi.date().required(),
+  travel_reason: Joi.string().required(),
+  accomodationId: Joi.number().required(),
+});
+
+class tripRequestValidations {
+  static async validateTrip(req, res, next) {
+    try {
+      const wholeTrip = req.body;
+
+      // check if accomodation is in the same city with location
+      const acc = await Accommodation.findOne({
+        where: { id: wholeTrip.accomodationId },
+      });
+
+      if (!acc) {
+        return errorResponse.notFoundError(res, 'Accommodation not found');
+      }
+
+      if (acc.locationId !== wholeTrip.destination) {
+        return res.status(400).json({
+          message: `Accommodation must be in destination city`,
+        });
+      }
+
+      // departure place must not be the same as destination
```

Comment:
Why the comment ?

Reply:
this file has a lot of functionality, so to make it clear i left comments.